scripts/fund-pause-proxy.sh

```diff
@@ -0,0 +1,26 @@
+#!/usr/bin/env bash
+set -e
+
+CHANGELOG="0xdA0Ab1e0017DEbCd72Be8599041a2aa3bA7e740F"
+NETWORK="goerli"
+VALUE="1ether"
```

Comment:
We'll want to make this a smaller value on mainnet, or better, require an arg param to avoid foot-shooting.

Reply:
For sure, don't want someone to test run this and blow 1 ETH over every time.  What should it be. 0.1 ETH for mainnet?

Reply:
could also add a check that doesn't send it if the PP already has X amount